helpers/image_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require 'tempfile'
+require_relative '../config/StaticData'
+
+# Class for working with images after convert service
+class ImageHelper
+  def self.get_image_size(url)
+    Tempfile.create('images-', StaticData::TMP_FOLDER.to_s) do |fo|
```

Comment:
Previous note about TMP_FOLDER not need at all is still actual

Reply:
ok, i got it

Reply:
You can leave no reply in case you're agree with comment, just fix it :)
Github automatically mark comments of code that changes  as `resolved`